composer.json

```diff
@@ -59,7 +59,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.x-dev"
+            "dev-master": "1.7.x-dev"
```

Comment:
I think that this should not be changed.

Reply:
I'm simply reverting a change that predates the creation of the `2.x` branch, which is a Composer-friendly name. `master` is stil on the 1.x for now, until we merge the `2.x` onto it.

Reply:
That's a interesting point, I don't know if we'll merge 2.x once the work on this branch will be over.

Anyway, I'm still confused about the `1.7.x-dev` since we are on the 2.x branch.

And even if we use `1.x` as a branch alias, shouldn't it be `1.9.x`? The last version is `1.9.2`: https://packagist.org/packages/liip/functional-test-bundle